addons/purchase/data/mail_template_data.xml

```diff
@@ -49,8 +50,56 @@
             with reference: ${object.partner_ref}
         % endif
         amounting in <strong>${format_amount(object.amount_total, object.currency_id)}</strong>
-        from ${object.company_id.name}.
+        from ${object.company_id.name}. 
+        % if object.date_planned or object.expected_date:
+            The receipt is planned for 
+            % if object.date_planned:
+                <strong>${object.date_planned.date()}</strong>.
+            % elif object.expected_date:
+                <strong>${object.expected_date.date()}</strong>.
+            % endif
+            <br/><br/>
+            Could you please acknowledge the receipt of this order?
+            <!-- <br/><br/>
+            <div style="text-align: center;">
+                <a href="/my/purchase/${object.id}/confirm_po" class="btn btn-alpha btn-lg" style="text-align: center;">Confirm</a>
+            </div> -->
```

Comment:
to clear up

Reply:
About this button, @oma-odoo want me to hide this button in the mail layout, like the one we already have in sales and purchase, But those are only for view the order, they don't contain logic for extra action that can modify our record. But this button can do the extra comfirm action, and the user even don't know they will send a mail with that button, they may modify the content of the mail make it has nothing to do with the confirmation, but the button will still be there.  So I guess it's better to expose this button to the user. What do you think?

Reply:
i think your concern is legit but we should be consistent with sale and they seems to hide this button in the editor